src/fides/api/db/seed.py

```diff
@@ -327,7 +327,7 @@ def load_default_dsr_policies() -> None:
         excluded_data_categories = [
             "user.financial",
             "user.payment",
-            "user.credentials",
+            "user.authorization.credentials",
```

Comment:
having a bit of trouble reconciling this update with what i understand [this comment](https://github.com/ethyca/fides/pull/3933/files#r1299152122) to mean - just would like to confirm this is going to work as we expect

Reply:
@ThomasLaPiana just want to confirm here that we're OK having the default policy include the second-level `user.authorization`, because that's effectively what we're going to end up with here. and then i'm pretty sure this means the default policy will, in effect, also target `user.authorization.credentials`, since we include the children of data category targets when filtering DSR results, as far as i can tell ([see `filter_results.filter_data_categories`](https://github.com/ethyca/fides/blob/main/src/fides/api/task/filter_results.py#L65)). so this seems undesirable.

my suggestion is that we either update this to exclude `user.authorization` (simpler update, but not sure if it's exactly the behavior we're looking for) or we update the logic of `filter_data_categories` here in `seed.py` to change the second-level logic (more complex update that i think would deserve some more thought/discussion). simple code suggestion below for the former:
```suggestion
            "user.authorization",
```

Reply:
I think we want `user.authorization.credentials` to match what we had before, but I primarily made this change with find/replace, I'm not sure what broader implications might be

Reply:
OK, but the update here effectively makes this line a no-op -- `user.authorization.credentials` isn't included in the default policy regardless, because it is 3 levels deep, and we explicitly only include categories [that are `< 3` levels within `seed.filter_data_categories`](https://github.com/ethyca/fides/blob/ThomasLaPiana-update-fideslang-200/src/fides/api/db/seed.py#L128).

but more than just being a no-op, we're actually going to construct a default access policy here that _does_ include `user.authorization` as a target data category - i've confirmed that locally by looking at the default policy that's created. by including `user.authorization` as a target data category, we also effectively target `user.authorization.credentials` in that policy, so we're effectively changing the behavior here to now _include_ user credentials data in our default policy. i don't think that change in behavior is intended.

my suggestion is to _not_ include `user.authorization` as a target data category in the default access policy. this will effectively keep the existing behavior of _not_ including user credentials data in our default access policy. the only additional side effect is that we'd also be excluding any _other_ `user.authorization` data from the access policy, i.e. any fields tagged broadly with `user.authorization` or with its other subcategories, `user.authorization.biometric`, `user.authorization.password`. IMO those side effects are OK, and worth keeping in-tact the behavior to not include user credentials in our default access policy (though that judgement is a bit subjective).

if there aren't any objections to what i said, then i'll go ahead and commit this update!

Reply:
ah, ok, I agree with you @adamsachs I think we should exclude the higher-level `user.authorization`, which seems to be in line with the intent of that `excluded_data_categories` section, meaning any data category or nested data category in that list will _not_ be returned in the DSR package and _not_ erased by default.

Follow up then, it seems like we need to include a migration of existing RuleTargets on the default access/erasure policies.

Reply:
oh good point, i'd think just updating _all_ `RuleTargets` would probably be most consistent with the migration/upgrade behavior we're going for here...

Reply:
Updated as part of the migration and merged into this branch